[2025-10-31T02:29:11.557+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-31T02:29:11.588+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_postgres_dag.extract_api_data manual__2025-10-31T02:29:09.958149+00:00 [queued]>
[2025-10-31T02:29:11.596+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_postgres_dag.extract_api_data manual__2025-10-31T02:29:09.958149+00:00 [queued]>
[2025-10-31T02:29:11.597+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-10-31T02:29:11.610+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_api_data> on 2025-10-31 02:29:09.958149+00:00
[2025-10-31T02:29:11.618+0000] {standard_task_runner.py:63} INFO - Started process 654 to run task
[2025-10-31T02:29:11.621+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'api_to_postgres_dag', 'extract_api_data', 'manual__2025-10-31T02:29:09.958149+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/api_to_postgres_dag.py', '--cfg-path', '/tmp/tmp9v05u85b']
[2025-10-31T02:29:11.624+0000] {standard_task_runner.py:91} INFO - Job 30: Subtask extract_api_data
[2025-10-31T02:29:11.683+0000] {task_command.py:426} INFO - Running <TaskInstance: api_to_postgres_dag.extract_api_data manual__2025-10-31T02:29:09.958149+00:00 [running]> on host 754154eb0275
[2025-10-31T02:29:11.774+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='lenny' AIRFLOW_CTX_DAG_ID='api_to_postgres_dag' AIRFLOW_CTX_TASK_ID='extract_api_data' AIRFLOW_CTX_EXECUTION_DATE='2025-10-31T02:29:09.958149+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-31T02:29:09.958149+00:00'
[2025-10-31T02:29:11.775+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-31T02:29:11.903+0000] {logging_mixin.py:188} INFO - âœ… Extracted and saved raw data.
[2025-10-31T02:29:11.904+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-31T02:29:11.905+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-31T02:29:11.917+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=api_to_postgres_dag, task_id=extract_api_data, run_id=manual__2025-10-31T02:29:09.958149+00:00, execution_date=20251031T022909, start_date=20251031T022911, end_date=20251031T022911
[2025-10-31T02:29:11.955+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-31T02:29:11.980+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-31T02:29:11.983+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
